<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="add_node_Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Node</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="enter_First_nodeName_edge_TextField">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="enter_Second_nodeName_edge_TextField">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="make_edge_Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Edge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Node 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Node 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="dictionary_viewer_TextEdit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>370</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="visualize_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>741</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bfs_radioButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BFS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dijkstra_radioButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dijkstra's</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ballman_ford_radioButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>260</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Bellman-Ford</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dfs_radioButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DFS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allow_weight_Button">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>140</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow 
Weights</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edge_weight_TextField">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dictionary</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="algorithms_results_viewer_TextEdit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>580</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>545</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithms Search Results</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
